cmd/lilypad/collection-pow-submission.go

```diff
@@ -0,0 +1,50 @@
+package lilypad
+
+import (
+	"github.com/lilypad-tech/lilypad/pkg/collectpowsubmission"
+	"github.com/lilypad-tech/lilypad/pkg/options"
+	optionsfactory "github.com/lilypad-tech/lilypad/pkg/options"
+	"github.com/lilypad-tech/lilypad/pkg/system"
+	"github.com/lilypad-tech/lilypad/pkg/web3"
+	"github.com/spf13/cobra"
+)
+
+func collectionPowSubmissionCmd() *cobra.Command {
```

Comment:
Could we follow the existing pattern for naming commands and call this `newCollectionPowSubmissionCmd`? Event better, we would like to start using the convention for initialisms/acronyms mentioned here: https://go.dev/wiki/CodeReviewComments#initialisms, which would change the name to `newCollectionPOWSubmissionCmd`.

Reply:
done